scripts/enable-backend.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+# This script will enable the backend for an customer instance.
+# All needed credentials are loaded/stored from/to the Bitwarden Secrets Manager
+
+# how to use
+#
+# make sure to install the dependencies: ./install-dependencies.sh
+#
+# ./enable-backend.sh <instance>
+# example: ./enable-backend.sh qm
+#
+# Attention: on macos, see setEnv function and enable the macos line instead the linux line
+#
+
+##############################
+# setup
+##############################
+
+source "../setup.env"
+
+# check if BWS_ACCESS_TOKEN is set
+if [[ -z "${BWS_ACCESS_TOKEN}" ]]; then
+  echo "BWS_ACCESS_TOKEN is not set. Abort."
+  exit 1
+fi
+
+# set server-base to EU instance
+bws config server-base https://vault.bitwarden.eu
+
+##############################
+# ask for input data
+##############################
+
+if [ -n "$1" ]; then
+  instance="$1"
+else
+  echo "What is the name of the instance?"
+  read -r instance
+fi
+
+##############################
+# variables
+##############################
+
+path="../../$PREFIX$instance"
+
+# load secrets from Bitwarden Secret Manager
+RENDER_API_CLIENT_ID_DEV=$(bws secret -t "$BWS_ACCESS_TOKEN" get "b53d7a1d-220e-4e07-b1f9-b22700711f79" | jq -r .value)
+RENDER_API_CLIENT_SECRET_DEV=$(bws secret -t "$BWS_ACCESS_TOKEN" get "83a8e38b-fc22-461f-91a0-b22700712b62" | jq -r .value)
```

Comment:
we wouldn't have to add these by default unless `FEATURES_EXPORTAPI_ENABLED`. So this would be another enable-feature script, maybe?